app/client/src/components/shared/VirtualizedList.js

```diff
@@ -1,96 +1,89 @@
 // @flow
 
-import React, { useEffect, useRef, useState } from 'react';
-import {
-  AutoSizer,
-  CellMeasurer,
-  CellMeasurerCache,
-  List,
-  WindowScroller,
-} from 'react-virtualized';
+import React, { useCallback, useEffect, useRef } from 'react';
+import { useWindowSize } from '@reach/window-size';
+import { VariableSizeList } from 'react-window';
+import throttle from 'lodash/throttle';
 // utils
 import { useOnScreen } from 'utils/hooks';
 
```

Comment:
This could be pulled up to get rid of a couple 'already declared in the upper scope' code smells.
```suggestion
function RowRenderer({ index, renderer, setSize, width, listRef }) {
  const rowRef = useRef();

  // keep track of the height of the rows to autosize rows
  useEffect(() => {
    if (!rowRef?.current) return;

    setSize(index, rowRef.current.getBoundingClientRect().height, listRef);
  }, [setSize, index, width, listRef]);

  return <div ref={rowRef}>{renderer({ index })}</div>;
}

```

Reply:
Good catch. I went ahead and moved that function out into a separate component.